nodebb

```diff
@@ -2,36 +2,34 @@
 
 'use strict';
 
-var cproc;
-var args;
-var fs;
-var path;
-var request;
-var semver;
-var prompt;
-var async;
+var fs = require('fs');
+var path = require('path');
+var cproc = require('child_process');
 
+// check to make sure dependencies are installed
 try {
-	require('colors');
-	cproc = require('child_process');
-	args = require('minimist')(process.argv.slice(2));
-	fs = require('fs');
-	path = require('path');
-	request = require('request');
-	semver = require('semver');
-	prompt = require('prompt');
-	async = require('async');
+	fs.readFileSync(path.join(__dirname, 'node_modules/async/package.json'));
```

Comment:
Unfortunately we can't trust npm with that. Just today `request` wasn't properly installed for me, and I needed to reinstall it.